src/Glpi/Progress/ProgressStorage.php

```diff
@@ -112,7 +81,7 @@ public function save(SessionProgress $progress): void
 
         Session::start();
```

Comment:
I didn't notice in the original PR, but isn't it weird that a lot of methods in the file call `Session::start` ?
I would expect the session to already be initialized by our kernel (and to throw an exception if the sessions isn't available).

Reply:
It is mandatory to use the `session_write_close()` function to release the lock of the session file in the request that executes the process to be able to read it from the requests fetching the progress indicator info.

I added some code comments to explain this. I also removed the useless session start/close instructions on the `getCurrentProgress` method.